src/screens/HomeScreen.jsx

```diff
@@ -8,12 +8,13 @@ import handleLogOut from '../hooks/HandleLogOut';
 import OnAuthStateChanged from '../hooks/OnAuthStateChanged';
 import HomeScreenStyles from '../styles/HomeScreenStyles';
 import {useNavigation} from '@react-navigation/native';
-import Card from '../components/Card';
+import FlightList from '../components/FlightList';
 import AddButton from '../components/AddButton';
 
+
 const HomeScreen = () => {
   const user = OnAuthStateChanged();
-  const navigation = useNavigation();
+  const navigation = useNavigation();  
 
   const handleOnLogOut = async () => {
     console.log('handleOnLogOut executed');
```

Comment:
Don't leave debug code on GitHub, like console.log(), you can use Alert.alert() instead